### PR TITLE
Add unit tests to check empty(None) key fields (time,series,family) of query to RiakTS

### PR DESCRIPTION
**Pull request by AleksandrPavlenko**

### CODE REVIEW

File: connector/src/test/scala/com/basho/riak/spark/rdd/timeseries/TimeSeriesWriteTest.scala [hunk 1]
```diff
@@ -231,6 +231,75 @@ class TimeSeriesWriteTest extends AbstractTimeSeriesTest(false) with AbstractRDD
       """.stripMargin, stringify(data))
   }
 
+  @Test
+  def dataFrameWriteEmptyTime(): Unit = {
```
Comment (srgg) [inline connector/src/test/scala/com/basho/riak/spark/rdd/timeseries/TimeSeriesWriteTest.scala]: let's rename to bring more light on the expected behavior, kinda

dataFrameFailOnWritingEmptyTime
